expressEngine/cms/product/success.js

```diff
@@ -1,4 +1,4 @@
 export default {
-  userRegistered: "You have registered successfully",
-  productRegistered: "Product registered successfully"
+  userRegistered: 'You have registered successfully',
```

Comment:
There is no need to use userRegistered variable here as it is already handled in /cms/user/success.js

Reply:
I just fixed eslint errors, my issue doesn't concern success handling